src/gdal.mk

```diff
@@ -87,4 +87,10 @@ define $(PKG)_BUILD
     $(MAKE) -C '$(1)/ogr'   -j '$(JOBS)' install OGR_ENABLED=
     $(MAKE) -C '$(1)/apps'  -j '$(JOBS)' install
     ln -sf '$(PREFIX)/$(TARGET)/bin/gdal-config' '$(PREFIX)/bin/$(TARGET)-gdal-config'
+
+    mkdir -p '$(PREFIX)/$(TARGET)/bin
```

Comment:
I don't think the `mkdir` is necessary in this case. If it was needed the symlink above wouldn't work :)